apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -41,6 +43,19 @@ class LessonOverview extends Component {
     hasVerifiedResources: PropTypes.bool.isRequired
   };
 
+  constructor(props) {
+    super(props);
+
+    analyticsReporter.sendEvent(EVENTS.LESSON_OVERVIEW_PAGE_VISITED_EVENT, {
+      lessonId: props.lesson.id,
+      lessonName: props.lesson.displayName,
+      lessonLink: document.location.pathname,
+      referrer: document.referrer,
+      unitName: props.lesson.unit.displayName,
+      unitLink: props.lesson.unit.link
```

Comment:
Just to make sure I'm following - there's no way to get props.lesson.unit.id here?

Reply:
No, unfortunately that's not included: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/lessonOverview/lessonPlanShapes.jsx#L72-L88. I'm not sure why.

Reply:
🤦🏻‍♀️ okay then for sure ✅

Reply:
realizing that link doesn't show what I thought it did, so here's a better reference
![image](https://user-images.githubusercontent.com/1382374/217940595-db5d1863-2142-4cb3-8628-49c81d5c5194.png)

Reply:
Honestly, I don't think we've ever needed it on this page before 🤷‍♀️ It would be pretty easy to pass down from the server if we wanted to do that (as a follow up)